src/postprocess/stresses.jl

```diff
@@ -26,12 +25,28 @@ for details. In Voigt notation one would use the vector
                                    basis.use_symmetries_for_kpoint_reduction,
                                    basis.comm_kpts, basis.architecture)
         ρ = compute_density(new_basis, scfres.ψ, scfres.occupation)
-        energies = energy_hamiltonian(new_basis, scfres.ψ, scfres.occupation;
-                                      ρ, scfres.eigenvalues, scfres.εF).energies
+        (; energies) = energy(new_basis, scfres.ψ, scfres.occupation;
+                              ρ, scfres.eigenvalues, scfres.εF)
         energies.total
     end
-    L = scfres.basis.model.lattice
-    Ω = scfres.basis.model.unit_cell_volume
-    stresses = ForwardDiff.gradient(M -> HF_energy((I+M) * L), zero(L)) / Ω
-    symmetrize_stresses(scfres.basis, stresses)
+    L  = scfres.basis.model.lattice
+    Ω  = scfres.basis.model.unit_cell_volume
+    stresses = 1/Ω * ForwardDiff.gradient(zeros(eltype(L), 6)) do M
+        D = [1+M[1]   M[6]   M[5];  # Lattice distortion matrix
+               M[6] 1+M[2]   M[4];
+               M[5]   M[4] 1+M[3]]
+        HF_energy(D * L)
+    end
+    symmetrize_stresses(scfres.basis, voigt_to_full(stresses))
+end
+function voigt_to_full(v::AbstractVector{T}) where {T}
+    [v[1]       v[6]/T(2)  v[5]/T(2);
+     v[6]/T(2)  v[2]       v[4]/T(2);
+     v[5]/T(2)  v[4]/T(2)  v[3]     ]
+end
```

Comment:
Why is there a factor of 2 here compared to the formula above?

Reply:
I added more comments and details.